Wiki pages on site

couven92 opened the pull request:
Quick and dirty preliminary integration of wiki pages into the air:bit website.

Should in the future be replaced with a git submodule to https://github.com/skolelab/airbit.wiki.git instead with appropiate modifications to the Dockerfile to generate the wiki pages at Docker-build.

fjukstad commented:
Looks good. Please fix these before I merge: 

1. Modify the `views/index.html`so that the "Programmering" does not open in a new window/tab. (Remove the `target="_blank"` in the link)
2. Either remove the "Tekniske Spesifikasjoner" link in the menu or create an empty page so that clicking the link does not result in a `404 page not found`.
3. Add some styling (e.g. `max-width`) to the images to scale down the images to fit the width of the web page. See the image below

![screen shot 2018-01-03 at 12 44 15](https://user-images.githubusercontent.com/1394111/34519397-0cfee6f0-f084-11e7-88c3-889e6fca0da6.png)

couven92 commented:
@fjukstad Okay, fixed temporarily

I seem to have lost the pages with the technical specifications, so I'll have to recreate them, but in the meantime we should upload this to the live site.

I have modified my HTML generation script in https://github.com/skolelab/airbit.wiki.git to spit out a warning for unresolved links to prevent this from happening in the future.

fjukstad commented:
Cheers!
